sites/example-project/src/components/ui/Databases/GenericForm.svelte

```diff
@@ -5,9 +5,9 @@
 	export let credentials;
 	export let disableSave;
 
-	let requiredOpts = opts.filter((d) => d.optional !== true);
-	let optionalOpts = opts.filter((d) => d.optional === true);
-	let overrideOpts = opts.filter((d) => d.optional === true && d.override === true);
+	$: requiredOpts = opts.filter((d) => d.optional !== true);
+	$: optionalOpts = opts.filter((d) => d.optional === true);
+	$: overrideOpts = opts.filter((d) => d.optional === true && d.override === true);
 
```

Comment:
These variables should still be declared (e.g. with `let`)

Reply:
Why? The compiler inserts declarations for reactively declared variables.

Reply:
Oh that's neat I didn't know that